CMakeLists.txt

```diff
@@ -219,6 +222,8 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang|AppleClang")
   target_compile_options(celerity_runtime PRIVATE -Wall -Wextra -Wno-unused-parameter -Werror=return-type -Werror=init-self)
 endif()
 
+target_compile_options(celerity_runtime PRIVATE "${CELERITY_CXX_FLAGS}")
```

Comment:
Instead of adding `${CELERITY_CXX_FLAGS}` to a target manually in `add_celerity_to_target`, couldn't this just be made ` PUBLIC`?

Reply:
You're right!